feature-libs/product/configurators/common/core/state/effects/configurator-basic.effect.ts

```diff
@@ -58,6 +73,21 @@ export class ConfiguratorBasicEffects {
     })
   );
 
+  collectDisabledGroupIds(
```

Comment:
And why do we need this at all? Why can't we just check for the 'configurable' attribute in the groups?